assets/css/styles.css

```diff
@@ -80,15 +80,164 @@ input {
     background-color: var(--dark-grey-color);
 }
 
-/* ---------- Home Page Styles ---------- */
```

Comment:
Hi @martin-co77 

Could you add your homepage styles into the Home Page Styles section here please, instead of removing it please